app/script/view_model/content/PreferencesDeviceDetailsViewModel.js

```diff
@@ -83,7 +86,7 @@ z.ViewModel.content.PreferencesDeviceDetailsViewModel = class PreferencesDeviceD
 
   _update_fingerprint() {
     this.cryptography_repository.get_remote_fingerprint(this.self_user().id, this.device().id).then(fingerprint => {
-      this.fingerprint(fingerprint);
+      this.fingerprint(z.util.zero_padding(fingerprint, 16));
```

Comment:
magic..

Reply:
The `16` isn't magic. It's the length:

![unbenannt](https://user-images.githubusercontent.com/469989/34516860-ed62ab7a-f078-11e7-97b3-7f47234ec87c.PNG)